lib/db/parsed-statement.js

```diff
@@ -21,10 +21,9 @@ function ParsedStatement(type, operation, collection, raw) {
   }
 }
 
-ParsedStatement.prototype.recordMetrics = function recordMetrics(segment, scope) {
+ParsedStatement.prototype.recordMetrics = function recordMetrics(segment, scope, transaction) {
```

Comment:
Do you think we will eventually remove `segment` from the parameters and then retrieve it from the transaction in the function body?